### PR TITLE
[Easy] Bump @truffle/contract

### PR DESCRIPTION
**Pull request by bh2smith**
A [bug was introduced](https://github.com/trufflesuite/truffle/issues/3024#event-3323111945) with the previous truffle/contract version so we told dependabot to ignore it. 

This is the most recent version. 

Does anyone (@nlordell) happen to know if we need to tell dependabot to start monitoring again?

### CODE REVIEW

**Comment by bh2smith** [comment]
Sorry, my branch was WAY behind master. Fixing ASAP.

File: types/truffle-typings/merge.d.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
+/* Generated by ts-generator ver. 0.0.8 */
```
Comment (fleupold) [inline types/truffle-typings/merge.d.ts:1]: Isn't this file now part of builds? I was under the impression we shouldn't been checking in the types that are generated.

**Review by fleupold — CHANGES_REQUESTED** [review]
Shouldn't this update some file in package.json? Also I think the auto-generated file should not be checked in to version control

File: types/truffle-typings/merge.d.ts (hunk 1 above)
Comment (bh2smith) [reply types/truffle-typings/merge.d.ts:1]: Ya, I don't know what this is. Sorry. The change was only supposed to be the version bump.

**Comment by bh2smith** [comment]
This version bump was already made.

**Pull request closed without merging**